diff --git a/node_modules/@modelcontextprotocol/sdk/dist/cjs/client/stdio.js b/node_modules/@modelcontextprotocol/sdk/dist/cjs/client/stdio.js
index ee48b40..93a5191 100644
--- a/node_modules/@modelcontextprotocol/sdk/dist/cjs/client/stdio.js
+++ b/node_modules/@modelcontextprotocol/sdk/dist/cjs/client/stdio.js
@@ -76,7 +76,7 @@ class StdioClientTransport {
                     ...this._serverParams.env
                 },
                 stdio: ['pipe', 'pipe', this._serverParams.stderr ?? 'inherit'],
-                shell: false,
+                node_process_1.default.platform === "win32",
                 windowsHide: node_process_1.default.platform === 'win32' && isElectron(),
                 cwd: this._serverParams.cwd
             });
diff --git a/node_modules/@modelcontextprotocol/sdk/dist/esm/client/stdio.js b/node_modules/@modelcontextprotocol/sdk/dist/esm/client/stdio.js
index e1e4b9c..73b10d4 100644
--- a/node_modules/@modelcontextprotocol/sdk/dist/esm/client/stdio.js
+++ b/node_modules/@modelcontextprotocol/sdk/dist/esm/client/stdio.js
@@ -69,7 +69,7 @@ export class StdioClientTransport {
                     ...this._serverParams.env
                 },
                 stdio: ['pipe', 'pipe', this._serverParams.stderr ?? 'inherit'],
-                shell: false,
+                shell: process.platform === "win32",
                 windowsHide: process.platform === 'win32' && isElectron(),
                 cwd: this._serverParams.cwd
             });
